### PR TITLE
formatted member count

### PR DESCRIPTION
**Pull request by Taz03**
* Format member count with proper commas
* For member count as `300000` it will be formatted to `300,000`(groups of thousand).
* update predicate to match category including commas in category name.

### CODE REVIEW

File: application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCountDisplayRoutine.java [hunk 1]
```diff
@@ -23,17 +24,16 @@ public class MemberCountDisplayRoutine implements Routine {
      */
     public MemberCountDisplayRoutine(Config config) {
         memberCountCategoryPredicate =
-                Pattern.compile(config.getMemberCountCategoryPattern() + "( - \\d+ Members)?")
+                Pattern.compile(config.getMemberCountCategoryPattern() + "( - [\\d,]+ Members)?")
```
Comment (ankitsmt211) [inline application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCountDisplayRoutine.java:27]: would be nice to write a comment for what this predicate would match as an example.

Comment (Taz03) [reply application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCountDisplayRoutine.java:27]: image

Comment (ankitsmt211) [reply application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCountDisplayRoutine.java:27]: i meant inside code itself 
//matches likes of `Info-1,000 Members`
or something like that

Comment (Taz03) [reply application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCountDisplayRoutine.java:27]: nah

Comment (ankitsmt211) [reply application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCountDisplayRoutine.java:27]: I can do it if it's a bother, problem with regex is it looks like magical stuff so you have to either use a third party tool like a platform to validate it which you don't need to if a comment mentions what sort of pattern we're trying to match which can ofc save everyone a bit of time and make it more readable.

**Review by christolis — APPROVED** [review]
LGTM

**Review by surajkumar — APPROVED** [review]

**Review by tj-wazei — APPROVED** [review]

**Pull request merged**